front/src/components/ViewMockup.js

```diff
@@ -0,0 +1,72 @@
+import React, {Component} from "react";
+import {Container} from 'reactstrap';
+import Fullscreen from "react-full-screen";
+import "../App.css";
+
+class ViewMockup extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            mockup: "",
+            isFull: false,
+            iframeWidth: '110%',
+            iframeHeight: window.innerHeight,
+            scale: 'scale(0.9)'
+        };
+        this.iframe = React.createRef();
+
+        fetch('http://localhost:8080/mockups/name/' + this.props.match.params.name)
+            .then(response => {
+                response.body.getReader().read().then(val => {
+                    const mockupString = new TextDecoder("utf-8").decode(val.value);
+                    const mockupObject = JSON.parse(mockupString);
+                    this.setState({
+                        mockup: mockupObject
+                    });
+                });
+            });
+    }
+
+    getSourceLink() {
+        return this.state.mockup.source;
+    }
+
+    getMockupName() {
```

Comment:
this method doesn't compile and is unused

Reply:
Deleted